File: Templates/Challenge/Hello_World/Compute/template.py
#!/opt/local/bin/python

#
# See ../README.md
#

#
# Run as: python ch-1.py [plain | compute]
#

import sys

COUNT   = 10
PLAIN   =  0
COMPUTE =  1

type = PLAIN

if len (sys . argv) > 1 and sys . argv [1] == "compute":
    type = COMPUTE


if type == PLAIN:
    print ("%TEXT")


if type == COMPUTE:
    1

File: Templates/Challenge/Hello_World/Compute/template.lua
#!/opt/local/bin/lua

--
-- See ../README.md
--

--
-- Run as: lua ch-1.lua [plain | compute]
--

local PLAIN   =  0
local COMPUTE =  1

local type = PLAIN
if   #arg >= 1 and arg [1] == "compute"
then type = COMPUTE
end

if   type == PLAIN
then print ("%TEXT")
end

if   type == COMPUTE
then
end
